src/components/Icons/ApplicationIcon/ApplicationIcon.tsx

```diff
@@ -1,90 +1,135 @@
-import { forwardRef, ForwardRefExoticComponent, RefAttributes } from 'react';
+import React, { FC } from 'react';
 
-// Needs to be relative path for the type to be importable after build
-import { AppIconProps } from '../../../types';
-import Acquire from './Acquire';
-import Bravos from './Bravos';
-import Dasha from './Dasha';
-import Fallback from './Fallback';
-import FourDInsight from './FourDInsight';
-import InPress from './InPress';
-import LoggingQualification from './LoggingQualification';
-import Orca from './Orca';
-import Portal from './Portal';
-import Premo from './Premo';
-import Pwex from './Pwex';
-import Recap from './Recap';
+import {
+  acquire,
+  amplify,
+  dasha,
+  embark,
+  equinor,
+  fallback,
+  ltg,
+  orca,
+  premo,
+  pwex,
+} from './ApplicationIconData/ApplicationIconCollection'; // Adjust imports as per your icons collection
+import { AllowedColors } from './ApplicationIcon.utils'; // Adjust the path as per your project structure
+import ApplicationBase from './ApplicationIconBase'; // Adjust the path as per your project structure
 
 export type ApplicationName =
-  | 'acquire'
-  | '4dinsight'
-  | 'recap'
+  | 'amplify'
+  | 'embark'
+  | 'premo'
   | 'dasha'
   | 'orca'
-  | 'portal'
-  | 'logging-qualification'
+  | 'acquire'
   | 'pwex'
-  | 'inpress'
-  | 'bravos'
-  | 'premo';
+  | 'ltg'
+  | 'equinor';
 
 interface ApplicationIconData {
   appName: string[];
```

Comment:
Is there a reason appName is an array of strings? Seems kinda weird, doesnt 1 icon only have 1 name?